src/main/java/org/galatea/starter/entrypoint/IexRestController.java

```diff
@@ -48,4 +52,16 @@ public List<IexLastTradedPrice> getLastTradedPrice(
     return iexService.getLastTradedPriceForSymbols(symbols);
   }
 
+  /**
+   * Get historical trade prices for each of the symbols passed in from a date passed in to present.
+   *
+   * @param from date that the prices start from
+   * @return a List of IexHistoricalPrices objects for the given symbols from a date.
+   */
+  @GetMapping(value = "${mvc.iex.getHistoricalPricesPath}", produces = {
+      MediaType.APPLICATION_JSON_VALUE})
+  public List<IexHistoricalPrices> getHistoricalPrices(@RequestParam(value = "from") @DateTimeFormat(pattern = "yyyy-MM-dd") final Date from,
+      @RequestParam(value = "symbols") final String symbols) {
```

Comment:
Is "symbols" a single symbol or a list of symbols?

Reply:
What happens if I don't supply a date?
What happens if I don't supply a symbol?
What happens if my symbol is an empty string?